subsys/net/ip/net_context.c

```diff
@@ -211,6 +177,7 @@ static int tcp_backlog_find(struct net_pkt *pkt, int *empty_slot)
 static int tcp_backlog_syn(struct net_pkt *pkt, struct net_context *context)
 {
 	int empty_slot = -1;
+	int rv;
```

Comment:
All the return values use the "ret" as variable name in this file, so lets be consistent about it here and change rv to ret.